docs/source/configuration/telemetry/instrumentation/instruments.mdx

```diff
@@ -209,14 +217,15 @@ telemetry:
 
 #### `value`
 
-The `value` of an instrument is the value which will be drawn from. This can be one of the following:
+The `value` of an instrument is the value which will be drawn from and is dependent on the stage that the instrument is defined at.
+This can be one of the following:
```

Comment:
The current definition of `value` is confusing and I'm not sure it adds anything. I think just the examples are enough to explain what it is.

```suggestion
An instrument's `value` is dependent on lifecycle stage that the instrument is defined at. `value` can be one of the following:
```